src/cascadia/TerminalApp/App.cpp

```diff
@@ -405,6 +405,13 @@ namespace winrt::TerminalApp::implementation
         _newTabButton.Flyout(newTabFlyout);
     }
 
+    void App::_OpenDropdown()
+    {
+        Controls::Primitives::FlyoutShowOptions options{};
+        options.Placement(Controls::Primitives::FlyoutPlacementMode::BottomEdgeAlignedLeft);
+        _newTabButton.Flyout().ShowAt(_newTabButton, options);

```

Comment:
Does this move UIA focus into the new tab dropdown? Should it? cc @carlos-zamora

Reply:
Good news: Xaml handles all of that for us as we change focus with the keyboard
"Bad" news: navigation between the flyout and its parent isn't great (but that's an existing issue). That'll get fixed in #2102 by me